packages/app/components/ConnectButton.tsx

```diff
@@ -0,0 +1,94 @@
+"use client";
+
+import { ConnectKitButton } from "connectkit";
+import { BodyText, Button } from "../ui";
```

Comment:
Why not use @/ui like in other files?

I think it's cleaner and it's not dependent where the file is.